ios/FluentUI/Command Bar/CommandBarItem.swift

```diff
@@ -9,7 +9,7 @@ public typealias CommandBarItemGroup = [CommandBarItem]
 
 @objc(MSFCommandBarItem)
 open class CommandBarItem: NSObject {
-    public typealias ItemTappedHandler = ((CommandBarItem) -> Void)
+    public typealias ItemTappedHandler = ((UIButton, CommandBarItem) -> Void)
```

Comment:
curious, does it need to know it is a button?

Reply:
No, just a UIView, I will update to reflect that, thanks!